cmd/report.go

```diff
@@ -346,3 +353,19 @@ func saveReport(baseName string, ext string, nodeFilter string, content string)
 	fmt.Printf("%s report saved to %s\n", strings.Title(baseName), reportPath)
 	return nil
 }
+
+func toAnonyimize() bool {
+
+	c, error := reporting.NewDefault()
+	if error != nil {
+		return false
+	}
+
+	if reportsFlags.anonymize {
```

Comment:
Could we collapse this into: 
`return reportsFlags.anonymize || c.Config.Reports.Anonymize`?

Reply:
Yep, done.